suricata.yaml.in

```diff
@@ -100,6 +104,18 @@ outputs:
       #facility: local5
       #level: Info ## possible levels: Emergency, Alert, Critical,
                    ## Error, Warning, Notice, Info, Debug
+      #redis:
```

Comment:
Just to check: redis support would be automatically work for all log outputs ,right? Not in favour of adding this block everywhere of course.